### ISSUE TITLE
Support for User Services

### ISSUE THREAD

**Issue by cemkeylan** [cemkeylan/sysmgr#1]
An idea that might just be scratched. Here is the deal,
we don't want root access to run user services. We also
don't want to run a second instance of sysmgr. Feedback
and ideas are welcome.

This is an idea for the future, `svctl` must be completely
stabilized before I get to this.

## What can be done RIGHT NOW to run user services

* Use a tool like sudo/su/chpst
  This can be done on the usual directory in services [requires root access].
* Run a second instance of `sysmgr`
  The user can change its environment variables to point
  to his/her home directory. This either has to be done
  manually everytime, or requires the first `sysmgr` instance
  to do it for them, which again requires root access.

These are messy ways to do it, here is why.

### A second instance of sysmgr

* If you do run two instances of sysmgr, you need to change
  your environment variables between calls to different services.
* Your home directory is filled with mess.
* You don't get the advantages of the `/run` directory, which is
  basically a temporary directory.
* The users could add a temporary directory for this purpose alone.
  But that would still require root access.

### Using tools on root services to run user services

This is how I am currently handling it. I have some services that
just do `chpst -u cem`.

## Ideas and questions for user services

* They are stored in `~/.config/sysmgr` or `~/.config/sysmgr/enabled`
* What is a reliable way of getting `HOME` and `XDG_CONFIG_HOME` variables for every non-system user?
* Am I thinking too much on "could I do this" that I disregard "should I do this" ?
